WebHostLib/templates/playerTracker.html

```diff
@@ -0,0 +1,176 @@
+{% block head %}
+    <!--suppress XmlDuplicatedId -->
+<title>{{ player_name }}&apos;s Tracker</title>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='styles/playerTracker.css') }}"/>
+    <script type="application/ecmascript" src="https://code.jquery.com/jquery-3.6.0.js"></script>
```

Comment:
Yeah, code scanner doesn't like this. Include an integrity check or include a distribution of this in the source to begin with (instead of loading from a CDN). Also @LegendaryLinux may have opinions as to whether we want to use jquery or not, I don't know; he's the pro.

Reply:
jQuery can go burn in a fire. I have already had to deal with the DataTables inclusion on some pages, but I will fight jQuery's inclusion anywhere else to the death. It's outdated, outperformed by native JS, and for some reason still taught to students as the "most popular" JS library.

Reply:
That is actually what I expected you to say, more or less.

Reply:
I only imported jquery as that was the only information I could find on dynamic information loading but was completely unable to get it to actually function so I'm fully for deleting that just need to get the tracker loading working.